### DISCUSSION TITLE
Welcome to Ruby 2D Discussions!

### DISCUSSION THREAD

**Discussion by blacktm** [ruby2d/ruby2d#186] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Souravgoswami** [comment]
Hello! Just wondering what's the plan with Ruby 3.0? Are we going to see any true multi-threading implementation that will make the app a lot faster? Or are we going to stick with the current implementation?

**Reply by blacktm** [reply]
We should keep an eye on [reactors in Ruby 3](https://github.com/ruby/ruby/blob/master/doc/ractor.md), which will offer true parallel execution.
